types/leaflet-boundary-canvas/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for leaflet-boundary-canvas 1.0
+// Project: https://github.com/ttungbmt/leaflet-boundary-canvas#readme
+// Definitions by: TurtIeSocks <https://github.com/TurtIeSocks>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as L from 'leaflet';
+import type { FeatureCollection } from 'geojson';
+
+declare module 'leaflet' {
+    namespace TileLayer {
+        interface BoundaryCanvasOptions extends TileLayerOptions {
+            boundary?: FeatureCollection;
+            crossOrigin?: boolean;
+            trackAttribution?: boolean;
+        }
+
+        class BoundaryCanvas extends TileLayer {
+            constructor(url: string, options?: BoundaryCanvasOptions);
+            static createFromLayer(tileLayer: TileLayer, options: BoundaryCanvasOptions): BoundaryCanvas;
```

Comment:
```suggestion
            static createFromLayer(tileLayer: TileLayer, options?: BoundaryCanvasOptions): BoundaryCanvas;
```
can this be undefined?, to just get an exact copy of the tile layer? (as the implementaiton augments `tileLayer.options` with anything passed as options)